pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2282,3 +2282,45 @@ type StorageClassDeviceSet struct {
 	// +optional
 	Encrypted bool `json:"encrypted,omitempty"`
 }
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CephFilesystemSubVolumeGroup represents a Ceph Filesystem SubVolumeGroup
+// +kubebuilder:subresource:status
+type CephFilesystemSubVolumeGroup struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	// Spec represents the specification of a Ceph Filesystem SubVolumeGroup
+	Spec CephFilesystemSubVolumeGroupSpec `json:"spec"`
+	// Status represents the status of a Ceph Client
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +optional
+	Status *CephFilesystemSubVolumeGroupStatus `json:"status,omitempty"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CephFilesystemSubVolumeGroup represents a list of Ceph Clients
+type CephFilesystemSubVolumeGroupList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+	Items           []CephFilesystemSubVolumeGroup `json:"items"`
+}
+
+// CephFilesystemSubVolumeGroupSpec represents the specification of a Ceph Filesystem SubVolumeGroup
+type CephFilesystemSubVolumeGroupSpec struct {
+	// VolName is the name of Ceph Filesystem SubVolumeGroup volume name, typically it's the name of
+	// the CephFilesystem CR.
```

Comment:
Should we explain how this value should be filled in when it's not given the name of a CephFilesystem CR? (and also propagate to the docs?)

Reply:
Yes I can add this.